packages/vuetify/src/stylus/settings/_variables.styl

```diff
@@ -311,13 +311,16 @@ $timeline-wedge-size := 10px
 $timeline-dot-small-size := 24px
 $timeline-dot-regular-size := 38px
 $timeline-dot-large-size := 52px
+$timeline-dot-medium-size := 48px
 
 $timeline-inner-dot-small-size := 18px
 $timeline-inner-dot-small-margin := 3px
 $timeline-inner-dot-regular-size := 30px
 $timeline-inner-dot-regular-margin := 4px
 $timeline-inner-dot-large-size := 42px
 $timeline-inner-dot-large-margin := 5px
+$timeline-inner-dot-medium-size := 36px
+$timeline-inner-dot-medium-margin := 4px
```

Comment:
Order these ascending (small, regular, medium, large)

Reply:
@KaelWD fixed in 91f84c3